### PR TITLE
IWF-124: Ignore empty output for collector

### PR DESCRIPTION
**Pull request by lwolczynski**
### Checklist
- [ ] Code compiles correctly
- [ ] Tests for the changes have been added
- [ ] All tests passing
- [x] **This PR change is backwards-compatible**
- [ ] **This PR CONTAINS a (planned) breaking change** (it is not backwards compatible)

### Related Issue
Closes #issue_number

### CODE REVIEW

File: service/interpreter/outputCollector.go [hunk 1]
```diff
@@ -7,16 +7,29 @@ type OutputCollector struct {
...
+			if output.CompletedStateOutput != nil {
+				filteredOutputs = append(filteredOutputs, output)
+			}
+		}
+
+		return &OutputCollector{
+			outputs: filteredOutputs,
+		}
 	}
 }
```
Comment (lwolczynski) [inline service/interpreter/outputCollector.go:27]: @longquanzheng is there a reason why we wouldn't filter out outputs with nil CompletedStateOutput here?

Comment (lwolczynski) [reply service/interpreter/outputCollector.go]: Discussed

**Review by longquanzheng — APPROVED** [review]
lgtm, 
do we know we have existing test with output(we probably do)
if no, can we change one of the test to return a non-nil output so that we can test the case?

**Comment by lwolczynski** [comment]
> lgtm, do we know we have existing test with output(we probably do) if no, can we change one of the test to return a non-nil output so that we can test the case?

basic_test.go:
`	assertions.Equal(1, len(resp2.GetResults()))`

conditional_close_test.go:
`	assertions.Equal(1, len(resp2.GetResults()))`
`assertions.Equal(expectedOutput, resp2.GetResults()[0])`

skip_start_test.go:
`	assertions.Equal(1, len(resp2.GetResults()))`

**Pull request merged**